herm_full_edgelist.csv

```diff
@@ -7377,4 +7376,20 @@ vm2pL , vm2pR       ,4,electrical
 vm2pR , VC03        ,2,electrical
 vm2pR , um1pR       ,4,electrical
 vm2pR , vm1pR       ,4,electrical
-vm2pR , vm2pL       ,4,electrical
```

Comment:
@lungd Why are these changes being added to herm_full_edgelist.csv?

Surely this should remain identical to https://github.com/openworm/PyOpenWorm/blob/dev/OpenWormData/aux_data/herm_full_edgelist.csv (or the reason for the change documented...)

Reply:
@pgleeson -- Is there a way to do this via code?  @lungd thinks we may have to add to the API to allow this?